client/my-sites/marketing/connections/service-description.jsx

```diff
@@ -236,6 +236,9 @@ class SharingServiceDescription extends Component {
 				} );
 			},
 			nextdoor() {
+				if ( this.props.numberOfConnections > 0 ) {
+					return this.props.translate( 'Sharing posts to your Nextdoor.' );
```

Comment:
I think it should be "to Nextdoor" or something like "to your Nextdoor profile".

Reply:
Done ✅